internal/node/node_repositories.bzl

```diff
@@ -29,28 +29,27 @@ _DOC = """To be run in user's WORKSPACE to install rules_nodejs dependencies.
 This rule sets up node, npm, and yarn.
 
 The versions of these tools can be specified in one of three ways:
-- Normal Usage:
+- Simplest Usage:
 Specify no explicit versions. This will download and use the latest NodeJS & Yarn that were available when the
 version of rules_nodejs you're using was released.
+Note that you can skip calling `node_repositories` in your WORKSPACE file - if you later try to `yarn_install` or `npm_install`,
+we'll automatically select this simple usage for you.
 - Forced version(s):
 You can select the version of NodeJS and/or Yarn to download & use by specifying it when you call node_repositories,
-but you must use a value that matches a known version.
+using a value that matches a known version (see the default values)
 - Using a custom version:
 You can pass in a custom list of NodeJS and/or Yarn repositories and URLs for node_resositories to use.
 - Using a local version:
 To avoid downloads, you can check in vendored copies of NodeJS and/or Yarn and set vendored_node and or vendored_yarn
-to point to those before calling node_repositories.
+to point to those before calling node_repositories. You can also point to a location where node is installed on your computer,
+but we don't recommend this because it leads to version skew between you, your coworkers, and your Continuous Integration environment.
```

Comment:
another reason we don't recommend is that it ties your build to a single platform as we don't support specifying multiple locations for different platforms.